src/backends/ilp-quoter/index.js

```diff
@@ -32,8 +30,10 @@ class ILPQuoter {
     const req = request({ method: 'PUT', uri, json: true })
     const res = yield req
     if (res.statusCode >= 400) {
-      throw new UnsupportedPairError(req.uri)
+      log.error('Unsupported pairs, the quoter reported an error on PUT ' + uri)
```

Comment:
Is there any way to distinguish between different types of quoter errors? Maybe log the res body to give the user more info.